.github/workflows/go.yaml

```diff
@@ -0,0 +1,62 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
What does "master" mean for pull requests? I mean the PRs are on it's own branch by definition.

Reply:
According to this, I understand that the syntax means the workflow is triggered when a PR is made that targets the master branch.
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags